canard.c

```diff
@@ -62,7 +62,7 @@
 struct CanardTxQueueItem
 {
     CanardTxQueueItem* next;
-    CanardCANFrame frame;
+    CanardTransportFrame frame;
```

Comment:
I am not really objecting here, just asking: is the renaming from `CanardCANFrame` to `CanardTransportFrame` intentional, or is it just a leftover from the previous PR? Since both protocols are variants of CAN, the old name would have been fine, too.

Reply:
It was intentional. I've started to refer to the "new" one as FD_FRAME. So now we got "CAN", "FD" and "TRANSPORT" for both. Do you have any suggestions for better naming?

Reply:
I think that when we say "CAN" we mean either "CAN 2.0" or "CAN FD", would you not agree? The taxonomy would be like this:

- Transport
  - CAN   <-- we are here
    - CAN 2.0B
    - CAN FD
  - UDP
  - IEEE 802.14